tests/ssl-opt.sh

```diff
@@ -1102,6 +1126,28 @@ is_gnutls() {
     esac
 }
 
+# Some external tools (gnutls or openssl) might not have support for ECDH and
```

Comment:
Minor: just to be sure it's clear that we're talking about ECDH as opposed to ECDHE, we should call it "static ECDH" to avoid the ambiguity.